components/FloatingMenu/floating.module.css

```
@@ -0,0 +1,187 @@
+.bars {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 4px;
+}
+
+.bars span {
+  display: block;
+  width: 18px;
+  height: 2px;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)